frontend/src/lib/components/common/skeleton/SkeletonElement.svelte

```diff
@@ -0,0 +1,11 @@
+<script lang="ts">
+	import { skeleton } from '../../../directives'
+	import { HEIGHT_UNIT, type ISkeletonElement } from './model'
+
+	export let element: ISkeletonElement
+</script>
+
+<div
+	use:skeleton={true}
```

Comment:
Couldn't we replay the whole use:skeleton with just the the three classes:
`classes="animate-pulse bg-blue-200 rounded"`

Reply:
Yes we could, but abstracting it with a directive will make it easier to update it if we ever need to.

Reply:
Everything is wrapped around an if statement in the Skeleton element. I don't think the code in the directive that removes the classes is ever reached.

Reply:
The SkeletonElement uses a constant `true` on the directive because only purpose of the component is to indicate loading. The toggling could be useful when the it's used directly. Sure it does the same as applying the classes in the class property but could be a little more useful in the future.

Reply:
I don't understand your comment @adam-kov